less-print/viewdetails.less

```diff
@@ -1,5 +1,94 @@
 .view-details-associations, .view-details-comments, .view-details-photos,
-.scroll-top-btn, .view-details-map, .location-static, .badge, .icons-header,
-.logo {
-  display: none;
+.scroll-top-btn, .view-details-map, .badge, .icons-header {
+ 	display: none !important;
+}
+
+.view-details-title {
+	font-size: .75em !important;
+	margin-top: 0px;
+	margin-bottom: 10px;
+}
+
+.view-details-info , .view-details-section .view-details-description {
+	float: none;
+}
+
+.view-details-section {
+	display: block;
+	font-size: .75em !important;
+	margin-left: 0;
+	padding-bottom: 0;
+	width: 100%;
+}
+
+.lead {
+	padding: 0 10px !important;
+}
+
+.heading {
+	padding: 5px 10px !important;
+}
+
+.view-details-section .view-details-informations .accordion {
```

Comment:
OK, I start another review only for compacting your code. With less and most other preprocessors, you can nest rules.
E.g., you could write lines 32 to 34 inside block at line 16 the following way:

    .view-details-section {
      display: block;
      font-size: .75em !important;
      margin-left: 0;
      padding-bottom: 0;
      width: 100%;
      
      .view-details-informations .accordion {
        padding: 0;
      }
    }